src/azureStorageExplorer/selectStorageAccountNodeForCommand.ts

```
@@ -46,19 +45,21 @@ export async function selectStorageAccountTreeItemForCommand(
     }
 
     if (options.mustBeWebsiteCapable) {
-        let hostingStatus = await accountTreeItem.getWebsiteHostingStatus();
+        let hostingStatus = await accountTreeItem.getActualWebsiteHostingStatus();
         await accountTreeItem.ensureHostingCapable(hostingStatus);
 
         if (options.askToConfigureWebsite && !hostingStatus.enabled) {
-            let result = await window.showInformationMessage(
-                `Website hosting is not enabled on storage account "${accountTreeItem.label}". Would you like to go to the portal to enable it?`,
-                DialogResponses.yes,
-                DialogResponses.no);
-            let enableResponse = (result === DialogResponses.yes);
+            let enableWebHostingPrompt = "Enable website hosting";
+            let result: { title: string } = await ext.ui.showWarningMessage(
```

You switched from `window.showInformationMessage` to `ext.ui.showWarningMessage`, which I think is a good thing, but it has different behavior. Specifically it'll throw `UserCancelledError` instead of returning undefined. Which means the logic below related to `cancelStep` will no longer work. Here's how I would fix:
```typescript
actionContext.properties.cancelStep = 'StorageAccountWebSiteNotEnabled';
actionContext.properties.enableResponse = 'false';
// no need to check result since cancel throws UserCancelledError and only other option is 'Enable'
await ext.ui.showWarningMessage(
    `Website hosting is not enabled on storage account "${accountTreeItem.label}".`,
    { modal: true },
    { title: "Enable website hosting" });
actionContext.properties.enableResponse = 'true';
await accountTreeItem.configureStaticWebsite();
// Either way can't continue
throw new UserCancelledError();
```

(Also note you don't have to pass in `DialogResponses.cancel` - that's always included by default)